crates/sui-protocol-config/src/lib.rs

```diff
@@ -4180,6 +4193,54 @@ impl ProtocolConfig {
         }
     }
 
+    pub fn binary_config(
```

Comment:
Should be `to_binary_config`, I believe.

Reply:
From #23963, but it matches the name of `verifier_config`